arches_component_lab/src/arches_component_lab/widgets/types.ts

```diff
@@ -13,7 +13,7 @@ export interface NewResourceInstance {
 }
 
 export interface ResourceInstanceReference {
-    resourceId: string;
+    resource_id: string;
     ontologyProperty: string;
     resourceXresourceId?: string;
     inverseOntologyProperty: string;
```

Comment:
I tried this to agree with arches-modular-reports 
```suggestion
    resource_id: string;
    display_value: string;
```
and still got errors:
```
arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/components/ResourceInstanceMultiSelectWidgetEditor.vue:73:17 - error TS2353: Object literal may only specify known properties, and 'ontologyProperty' does not exist in type 'ResourceInstanceReference'.

73                 ontologyProperty: "",
                   ~~~~~~~~~~~~~~~~

arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceSelectWidget/components/ResourceInstanceSelectWidgetEditor.vue:70:17 - error TS2353: Object literal may only specify known properties, and 'ontologyProperty' does not exist in type 'ResourceInstanceReference'.

70                 ontologyProperty: "",
                   ~~~~~~~~~~~~~~~~


Found 2 errors in 2 files.
```